proto/sentry_protos/snuba/v1/trace_item_attribute.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+package sentry_protos.snuba.v1;
+
+message AttributeKey {
+  enum Type { //this mostly reflects what types are able to be ingested, see eap_spans consumer for ingest details
+    TYPE_UNSPECIFIED = 0; //called "none" sometimes
+    TYPE_BOOLEAN = 1;
+    TYPE_STRING = 2;
+    TYPE_FLOAT = 3;
+    TYPE_INT = 4; //note: all numbers are stored as float64, so massive integers can be rounded. USE STRING FOR IDS.
+  }
+
+  Type type = 1;
+
+  //if you use span_id this will route to span.span_id,
```

Comment:
comment should be updated RE the namespacing, "if you use sentry.span_id, this will route to span.span_id, if you use derp this will route to ..."

Reply:
Would `duration` now be `sentry.duration`?

Reply:
yes, we're namespacing everything that isn't user sent to `sentry.` - that way the sentry backend can choose how to alias things if there are conflicts